heartbeat/_meta/beat.reference.yml

```diff
@@ -235,11 +234,1150 @@ heartbeat.monitors:
     # Interval between file file changed checks.
     #interval: 5s
 
-
 heartbeat.scheduler:
   # Limit number of concurrent tasks executed by heartbeat. The task limit if
   # disabled if set to 0. The default is 0.
   #limit: 0
 
   # Set the scheduler it's timezone
   #location: ''
+
+#================================ General ======================================
```

Comment:
I'm trying to understand where this huge diff here is coming from. Was that just missing before?

Reply:
Good question. I edited `beat.reference.yml` and copied it over to `_meta/beat.reference.yml`, which shouldn't have caused this. Maybe it was out of sync for a while?

Reply:
AFAICT this is good stuff to have in there at any rate right?

Reply:
@ruflin I fixed this file, this shouldn't have made it into this PR.